src/Input/FormField.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bemClassNames } from '../utils';
+import './formfield.css';
+
+const bem = bemClassNames('d2ui-form-field');
+
+const FormField = ({
+    children,
+    valid,
+    warning,
+    disabled,
+    error,
+    dense,
+    fullWidth,
+    helpText,
```

Comment:
I might be going overboard here, but following the MD they call the different labels:

* Label text
* Assistive text

The first is translated to `label` in property form, so I'm thinking we might do the same with the second: `assistive` (annoying to write) or `assist`. Then I note that @joeDHIS  calls it "Helper text", so maybe just `helper` is sufficient? Or `info`?

I vote for `helper`.

Reply:
I'd like to keep `helpText` here because `helper` is often used to describe utility functions, so I am afraid users might start passing functions into it. At the same time, I think that `info` is not descriptive enough. It works for a default state, because then that is what it is: some text that explains what to do with the input. However, once the component goes into _error_ or _warning_ state, then actually the text is there to **help** the user correct its input.

I just think that helpText is wonderfully apt for this situation:
- It is **text**
- It **helps** the user fill in the input correctly.

Reply:
implemented as `helperText`